packages/components/src/ProgressBar/Sizes.css

```diff
@@ -1,11 +1,11 @@
 .smaller {
-  --progress-bar--height: var(--space-smaller);
+  --progress-bar--height: 4px;
```

Comment:
Just following our desired pattern of not defining height/width dimensions with `space` tokens because if we increased "spacing" we would not want progress bars to resize